plugins/builds/create.js

```diff
@@ -16,18 +18,59 @@ module.exports = (server) => ({
             scope: ['user']
         },
         handler: (request, reply) => {
-            const Build = new Model.Build(
+            const build = new Model.Build(
```

Comment:
Yeah, that's because it returns an object.  You can do this, but since we're refactoring anyway, it's okay to skip.